txwinrm/util.py

```
@@ -112,9 +112,12 @@ def stopProducing(self):
 def _parse_error_message(xml_str):
     if not xml_str:
         return ""
-    elem = ET.fromstring(xml_str)
-    text = elem.findtext('.//{' + c.XML_NS_SOAP_1_2 + '}Text').strip()
-    detail = elem.findtext('.//{' + c.XML_NS_SOAP_1_2 + '}Detail/*/*').strip()
+    try:
+        elem = ET.fromstring(xml_str)
+        text = elem.findtext('.//{' + c.XML_NS_SOAP_1_2 + '}Text').strip()
+        detail = elem.findtext('.//{' + c.XML_NS_SOAP_1_2 + '}Detail/*/*').strip()
+    except:
```

Should at least be "except Exception". Never use a bare except as it even catches things like KeyboardInterruptError.